opaque_keys/edx/locator.py

```diff
@@ -1330,3 +1330,67 @@ def to_deprecated_list_repr(self):
 
 # Register AssetLocator as the deprecated fallback for AssetKey
 AssetKey.set_deprecated_fallback(AssetLocator)
+
+
+class AggregateCourseLocator(AggregateCourseKey):    # pylint: disable=abstract-method
```

Comment:
What formats of key does this need to parse? Can you add some tests to make sure it parses/serializes the way you expect?

Also, should you be able to get an `AggregateCourseKey` from a `CourseKey`?